jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -204,7 +206,7 @@ pipeline {
                                             parameters: [
                                                 string(name: 'TEST_MANIFEST', value: TEST_MANIFEST),
                                                 string(name: 'BUILD_MANIFEST_URL', value: buildManifestUrl),
-                                                string(name: 'AGENT_LABEL', value: AGENT_X64)
+                                                string(name: 'BUILD_MANIFEST_URL_OPENSEARCH', value: buildManifestUrlOpenSearch)
```

Comment:
Can you post the PR for integ-test job for dashboards which uses this change?
Kind of confused how and where this is used.